docs/theory/diffusion/intro.rst

```diff
@@ -0,0 +1,140 @@
+finds its target faster than can be explained with 3D diffusion alone. 
+Now, many experiments have shown that proteins often diffuse both in 1D diffusion (along DNA) and 3D diffusion (in the cytoplasm), also called facilitated diffusion :cite:`Berg1976, Berg1981`, which is more efficient that 3D diffusion alone.  
+
+.. image:: figures/target_search.png
+
+1D diffusion can can be directly visualized by holding a force-stretched DNA between optical tweezers and observing a fluorescently labeled protein bind and move along this DNA tether. 
+Pylake has many functions implemented for analyzing data from such an assay. This chapter focusses on the analysis of 1D diffusion of a protein on DNA.
+
+.. image:: figures/1D_diffusion.png
+
+
+
+The diffusion coefficient
+---------------------------
+
+A purely diffusive protein has the same probability to step to the right as to the left from any location on the DNA, therefore the average displacement is zero. 
+Rather than through velocity, diffusion is therefore quantified through the *diffusion coefficient (D)*. The diffusion coefficient quantifies how fast a protein explores its environment; 
+the units of the diffusion coefficients are :math:`length^{2}/s`. For 1D diffusion along DNA, D is often expressed in :math:`µm^{2}/s` or :math:`kbp^{2}/s` :cite:`Kochaniak2009`.
+
+Methods for obtaining the diffusion coefficient
+------------------------------------------------
+
+The first step for analyzing a diffusive protein from a kymograph, is to track each binding event. The resuling tracks give the positions of each bound protein over time.
+These tracks can then be analyzed further to determine the diffusion coefficient. Below we discuss two methods for diffusion analysis that are available in Pylake. 
+
+Mean Squared Displacement analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A common method for determining the diffusion coefficient, is to first compute the mean squared displacement (MSD) as a function of lag time for each track. 
+For purely diffusive motion, the first part of the MSD curve is linear, with the slope proportional to the diffusion coefficient. For 1D diffusion the MSD is given by: :math:`MSD = 2D\Delta t`.
+The diffusion coefficient can be obtained by fitting a linear function to the MSD curve, as illustrated in the figure below (simulated data):
+
+.. image:: figures/MSD_analysis.png
+
+Note that for the MSD curve, the 'lag time' (aka 'time interval') is given on the x-axis, rather than 'time'. 
+To compute the MSD for a certain lag time, you scan over a track, one data point at a time and compute the displacement squared 
+from that time until that time + the lag time. So each point on the MSD curve, is an average of displacements for a specific lag time, see the :doc:`theory section on diffusive motion</theory/diffusion/diffusion>`
+for the equation for the MSD. 
```

Comment:
It might be instructive to mention how this works. That for a track without gaps, you have `N-1` intervals to average over for `lag 1`, `N-2` for `lag 2` etc. To really drive home that you are averaging over time.